string part_name
float32[3] acceleration
float32[3] gyro
float32[3] magentometer
